hidimstat/standardized_svr.py

```diff
@@ -5,27 +5,56 @@
 from sklearn.pipeline import Pipeline
 
 
-def standardized_svr(X, y, Cs=np.logspace(-7, 1, 9), return_C=False, n_jobs=1):
+def standardized_svr(X, y, Cs=None, return_C=False, n_jobs=1):
+    """Cross-validated SVR
+
+    Parameters
+    -----------
+    X : ndarray or scipy.sparse matrix, (n_samples, n_features)
+        Data.
+
+    y : ndarray, shape (n_samples,) or (n_samples, n_targets)
+        Target. Will be cast to X's dtype if necessary.
+
+    Cs : ndarray, default=None
+        List of Cs where to compute the models.
+        If None, Cs are set automatically.
+
+    return_C : bool, default=False
```

Comment:
I think we should avoid arguments that change function signature. It makes function harder to understand and debug.